.github/workflows/orca.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           - INTEGRATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_DEV
         php-version: [ "8.1", "8.3" ]
         orca-enable-nightwatch: [ "FALSE" ]
-        orca-coverage-enable: [ "FALSE" ]
+        orca-coverage-enable: [ "TRUE" ]
```

Comment:
You probably told me why we're doing this, but I just want to call it out in case it was unintentional.

Reply:
I will leave a comment in the `orca.yml` file mentioning this.